docs/tutorials/config.md

```diff
@@ -24,15 +24,15 @@ Please refer to [mmcv](https://mmcv.readthedocs.io/en/latest/utils.html#config)
 We follow the below style to name config files. Contributors are advised to follow the same style.
 
 ```
-{model}_[model setting]_{backbone}_{neck}_[norm setting]_[misc]_[batch_per_gpu x gpu]_{schedule}_{dataset}
+{model}_[model setting]_{backbone}_[neck]_[norm setting]_[misc]_[batch_per_gpu x gpu]_{schedule}_{dataset}
```

Comment:
I'm not sure whether the neck is the required item when naming config files because some models have not neck part.

Reply:
Making it optional should be ok